### PR TITLE
feature: add new playground ui

### PR DESCRIPTION
**Pull request by anovazzi1**

### CODE REVIEW

File: src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                onKeyDown={(e) => {
+                  if (e.key === "Enter") {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    handleConfirm();
+                  }
+                }}
+                onChange={handleInputChange}
+                onBlur={(e) => {
+                  console.log(e.relatedTarget);
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx:123]: can we remove the console.log here?

File: src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      )}
+    >
+      <div className="flex w-full items-center justify-between gap-2 overflow-hidden px-2 py-1 align-middle">
+        <div className="flex min-w-0 items-center gap-2">
+          {isEditing ? (
+            <div className="flex items-center">
+              <Input
+                ref={inputRef}
+                value={editedSession}
+                onKeyDown={(e) => {
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx:114]: can we have a function to each interaction inside the HTML, to improve readability?
onKeyDown={handleInputKeyDown}

File: src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+                  if (e.key === "Enter") {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    handleConfirm();
+                  }
+                }}
+                onChange={handleInputChange}
+                onBlur={(e) => {
+                  console.log(e.relatedTarget);
+                  if (
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/IOFieldView/components/sessionSelector/newSessionSelector.tsx:124]: can we have a function to each interaction inside the HTML, to improve readability?

File: src/frontend/src/modals/IOModal/components/chatView/chatInput/newChatInput.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        />
+        <div className={classNameFilePreview}>
+          {files.map((file) => (
+            <FilePreview
+              error={file.error}
+              file={file.file}
+              loading={file.loading}
+              key={file.id}
+              onDelete={() => {
+                setFiles((prev: FilePreviewType[]) =>
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/chatInput/newChatInput.tsx:184]: can we have a function to each interaction inside the HTML, to improve readability?

File: src/frontend/src/modals/IOModal/components/chatView/chatInput/newChatInput.tsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  const fileInputRef = useRef<HTMLInputElement>(null);
+  const setErrorData = useAlertStore((state) => state.setErrorData);
+  const { validateFileSize } = useFileSizeValidator(setErrorData);
+
+  useFocusOnUnlock(lockChat, inputRef);
+  useAutoResizeTextArea(chatValue, inputRef);
+
+  const { mutate } = usePostUploadFile();
+
+  const handleFileChange = async (
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/chatInput/newChatInput.tsx:47]: **just a question!**
can we create a separate custom hook file to handle this, just to clean up the code in the component?

File: src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+    if (e.dataTransfer.files && e.dataTransfer.files.length > 0) {
+      handleFiles(e.dataTransfer.files, setFiles, currentFlowId, setErrorData);
+      e.dataTransfer.clearData();
+    }
+    setIsDragging(false);
+  };
+
+  const { mutate } = usePostUploadFile();
+
+  const handleFiles = (files, setFiles, currentFlowId, setErrorData) => {
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx:136]: **just a question!**
can we create a separate custom hook file to handle this, just to clean up the code in the component?

**Comment by ogabrielluiz** [comment]
@anovazzi1 Please, improve the PR title

**Review by Cristhianzl — APPROVED** [review]
lgtm

**Pull request merged**